packages/example-app-base/src/utilities/settings.ts

```diff
@@ -0,0 +1,19 @@
+import { getQueryParam } from './getQueryParam';
+
+/**
+ * Gets the value of a setting from the query or local storage.
+ * @param name Name of the setting (query param and/or session storage key)
+ * @returns The value, or empty string if not specified.
+ */
+export function getSetting(name: string): string {
+  let value = getQueryParam(name);
+  try {
+    // If there's not a value in the query, try getting it from session storage
+    value = value || sessionStorage.getItem(name) || '';
```

Comment:
any reason not to use localStorage? (you mention local storage in the description but session storage in implementation)

Reply:
I meant for it to be session storage all around. Mainly in case someone forgets they set it, so it doesn't stick around forever? Also because that's the pattern we already use with fabricVer.